codalab_service.py

```diff
@@ -203,7 +209,7 @@ def _get_parser():
                 '--version',
                 '-v',
                 type=str,
-                help='CodaLab version to use for building and deployment',
+                help='CodaLab version to use for building and deployment (defaults to branch name, set only for Travis CI)',
```

Comment:
```suggestion
                help='CodaLab version to use for building and deployment (defaults to branch name, needs to be set only for Travis CI)',
```

Otherwise it sounds like the default is set only on Travis CI (and not locally).